docs/advanced.md

```diff
@@ -67,7 +67,9 @@ The library uses 4 different logging levels:
 Check out this [example out for a live demonstration](../examples/custom-logging).
 
 ## Change the generated indentation type and size
-TODO 
+In some scenarios, depending on how you stitch them together, you might need to change the indentation type or size both of these cases are fully supported.
+
+Check out this [example out for a live demonstration](../examples/indentation-type-and-size).
```

Comment:
I have problem with that `live demonstration`. If I read live I think that there are some playground, but we have only source code to check, so please remove `live`. Please also remove that `live` in other parts of this doc.

Reply:
Please create another issue for this, regardless of the outcome, this won't be changed in this PR 🙂